modules/data.atmosphere/R/met.process.R

```diff
@@ -47,16 +47,76 @@ met.process <- function(site, input_met, start_date, end_date, model, host, dbpa
 
 
   #read in registration xml for met specific information
-  register.xml <- system.file(paste0("registration/register.", met, ".xml"), package = "PEcAn.data.atmosphere")
-  register <- read.register(register.xml, con)
-
+  met.reg.xml <- system.file(paste0("registration/register.", met, ".xml"), package = "PEcAn.data.atmosphere")
+  met.reg <- read.register(met.reg.xml, con)
+  
+ ##read in registration xml for model specific information
+ ## Where should these registration files be located? Within the model package? This could be difficult given that the package name might not be the same as model name in this context. Will need to double check. 
+  model.reg.xml <- system.file(paste0("registration/register.",model,".xml"), package = paste0("PEcAn.",model))
+  model.reg <- read.register(model.reg.xml, con) #### Betsy: rewrite read.register to accomodate met or model
+    
+  
+ ## Query Database for Input(Site,Source,Machine)
+ input.file.info = db.query(paste("SELECT * from inputs as i join dbfiles as d on i.id = d.container_id where i.site_id = ", 
```

Comment:
1) it seems to me that all the code below will break and throw an error if input.file.info returns NULL (i.e. a site that's never been run).
2) I don't see anywhere in this chunk of code that's checking the SOURCE of the data. If different products have the same format in their met.registry files then you may be processing update info on the entirely wrong product. For example, if I previously ran under CRUNCEP (format = netCDF CF) and now am trying to run a forecast under GFDL (format = netCDF CF) then this code will incorrectly grab the CRUNCEP record id's and mash GFDL met into CRUNCEP files. Furthermore, it appears that it wouldn't leave any trace that it did this.